lisp/bazel-util.el

```diff
@@ -27,7 +27,8 @@
 If FILE-NAME is not in a Bazel workspace, return nil.  Otherwise,
 the return value is a directory name."
   (cl-check-type file-name string)
-  (let ((result (locate-dominating-file file-name "WORKSPACE")))
+  (let ((result (or (locate-dominating-file file-name "WORKSPACE")
```

Comment:
This change looks useful, but seems unrelated. Could you please move it out into a separate PR?

Reply:
Moved to #80. Removed from this PR.